backend/core/Helpers/ExceptionHelper.cs

```diff
@@ -0,0 +1,80 @@
+using System;
+
+namespace Pims.Core.Helpers
+{
+    /// <summary>
+    /// ExceptionHelpers static class, provides methods to help with handling exceptions.
+    /// /// </summary>
+    public static class ExceptionHelper
+    {
+        /// <summary>
+        /// Provides a simple try+catch wrapper to set a variable.
+        /// </summary>
+        /// <param name="getValue"></param>
+        /// <param name="setter"></param>
+        /// <typeparam name="T"></typeparam>
+        /// <returns></returns>
+        public static T HandleKeyNotFound<T>(Func<T> getter, Func<T> setter)
```

Comment:
Just syntactical sugar so that we don't need ugly try+catch in various places to get/set values.

Reply:
Nice.